Childmonitoring.txt

```diff
@@ -5,7 +5,7 @@
 LiquidCrystal lcd(12, 11, 5, 4, 3, 2);

```

Comment:
1) Naming Convention: Variable names should be descriptive and use camelCase. Consider renaming DHTPIN and DHTTYPE to dhtPin and dhtType.
2) Commenting: Add comments to explain the purpose of the code and the functionality of each section.
3) Constants: Use 'const' keyword for constant values like pin numbers and sensor types to prevent accidental modification.
4) Whitespace: Add proper whitespace and indentation for better readability and consistency.
5) Include Libraries: Include necessary libraries for LiquidCrystal and DHT sensors at the beginning of the code.